src/sentry/features/temporary.py

```diff
@@ -608,6 +608,7 @@ def register_temporary_features(manager: FeatureManager) -> None:
     manager.add("projects:span-metrics-extraction", ProjectFeature, FeatureHandlerStrategy.INTERNAL, api_expose=True)
     manager.add("projects:span-metrics-extraction-addons", ProjectFeature, FeatureHandlerStrategy.INTERNAL, api_expose=False)
     manager.add("projects:relay-otel-endpoint", ProjectFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
+    manager.add("projects:relay-otel-logs-endpoint", ProjectFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
```

Comment:
The feature flag for the trace endpoint is now an organization level feature flag, should this be consistent, or do you need it per-project? https://github.com/getsentry/sentry/pull/99155

Reply:
Thanks for bringing this up, making it an organization level feature flag will be easier. Made the change here 56d90163d73e26e08e720e573ef48ddea73f0743